src/crimson/net/Socket.h

```diff
@@ -167,7 +167,7 @@ class FixedCPUServerSocket
   FixedCPUServerSocket(const FixedCPUServerSocket&) = delete;
   FixedCPUServerSocket& operator=(const FixedCPUServerSocket&) = delete;
 
-  seastar::future<bool> listen(entity_addr_t addr) {
+  seastar::future<> listen(entity_addr_t addr) {
```

Comment:
@cyx1231st looks great! could you squash this change into "crimson/net: implement FixedCPUServerSocket" ?

Reply:
squashed